src/main/java/hudson/plugins/copyartifact/TriggeredBuildSelector.java

```diff
@@ -95,21 +96,41 @@ public boolean isForGlobalSetting() {
         }
     };
     private Boolean fallbackToLastSuccessful;
-    private final UpstreamFilterStrategy upstreamFilterStrategy;
+    private UpstreamFilterStrategy upstreamFilterStrategy;
     private boolean allowUpstreamDependencies;
 
     @DataBoundConstructor
+    public TriggeredBuildSelector() {
+        this(false);
+    }
```

Comment:
Somehow the test written in #92 didn't failed even with `copyArtifacts(projectName: 'copiee', selector: upstream())`, it must have failed as `DataBoundConstroctor` with no parameters didn't exist.